### DISCUSSION TITLE
Using with Nuxtjs

### DISCUSSION THREAD

**Discussion by allaghi** [tailwindlabs/headlessui#363] [Help]
Hello everyone,
Have a nice day,

I am wondering if I could use it with nuxt js 

Thanks in advance.

**Comment by CodyBontecou** [comment]
I'm trying to get it working as well... I'll update this post if I can get it to work.

**Reply by CodyBontecou** [reply]
Oof! It didn't take long to realize it does not work `Please note that this library only supports Vue 3.` is posted at https://headlessui.dev/vue/menu. NuxtJS is currently Vue 2.

**Reply by allaghi** [reply]
Yes nuxtjs is currently Vue 2 , Thank you so much @CodyBontecou

**Reply by allaghi** [reply]
Do you think if nuxtjs upgrade to Vue 3 with nuxtjs 3 , I can use headless with it ?

**Reply by CodyBontecou** [reply]
For sure. I imagine the only reason it's not supported is due to Vue 3 rather than anything specific to Nuxt.
